src/revolve/ui/App.jsx

```
@@ -0,0 +1,733 @@
+  } catch (err) {
+    notification.error({
+      message: 'Connection Failed',
+      duration: 0,
+      style: {
+      width: 'auto',       // 👈 override the forced width
+      maxWidth: '90vw',    // 👈 or whatever you want
+      },
+      description: (
+        <div
+          dangerouslySetInnerHTML={{
+            __html: err.response?.data?.error || '<pre>Unable to reach the database.</pre>'
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)